frontend/mr-admin-flate/src/components/avtaler/OpprettAvtaleContainer.tsx

```diff
@@ -227,6 +227,23 @@ export function OpprettAvtaleContainer({
     );
   }
 
+  const ansvarligOptions = () => {
+    const options = [
+      {
+        value: ansatt?.ident ?? "",
+        label: `${navn} - ${ansatt?.ident}`,
+      }
+    ];
+
+    if (avtale?.ansvarlig && avtale.ansvarlig !== ansatt?.ident) {
+      options.push({
+        value: avtale?.ansvarlig,
+        label: avtale?.ansvarlig,
+      })
+    }
+    return options;
```

Comment:
Vi skal ikke snu om på disse? At det er den opprinnelige ansvarlige som står som ansvarlig, men at man også har valget for den man er logget inn som?

Reply:
Det er den opprinnelige som står som valgt nå. Selv om den er nummer to i lista. Men kan vel snu på dem uansett.